src/stories/connected-user.stories.tsx

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import '@stream-io/stream-chat-css/dist/css/index.css';
+import React, { useEffect, useState } from 'react';
+import { ChannelSort, StreamChat } from 'stream-chat';
+import {
+  Channel,
+  ChannelHeader,
+  ChannelList,
+  Chat,
+  MessageList,
+  useChannelStateContext,
+  Window,
+} from '../index';
+import { apiKey, StreamChatGenerics } from './utils';
+
+const channelId = import.meta.env.E2E_ADD_MESSAGE_CHANNEL;
+if (!channelId || typeof channelId !== 'string') {
+  throw new Error('expected ADD_MESSAGE_CHANNEL');
+}
+
+const Controls = () => {
+  const { channel } = useChannelStateContext();
+
+  return (
+    <div>
+      <button data-testid='truncate' onClick={() => channel.truncate()}>
+        Truncate
+      </button>
+      <button
+        data-testid='add-message'
+        onClick={() =>
+          channel.sendMessage({
+            text: 'Hello world!',
+          })
+        }
+      >
+        Add message
+      </button>
+    </div>
+  );
+};
+
+// Sort in reverse order to avoid auto-selecting unread channel
+const sort: ChannelSort = { last_updated: 1 };
+
+const chatClient = StreamChat.getInstance<StreamChatGenerics>(apiKey);
+
+const ConnectedUser = ({ token, userId }: { token: string; userId: string }) => {
```

Comment:
Took a look at this one and fixed it up a bit in Martin's branch [here](https://github.com/GetStream/stream-chat-react/compare/e2e...e2e-proposals) - other than that I think it's good to go. :taco:

Reply:
Thanks for cleaning this up. I was confused :D